<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biologic age</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,5,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="logo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>images/BME.jpg</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
        <item>
         <widget class="QLabel" name="sexLabel">
          <property name="text">
           <string>Sex:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sexCombo">
          <property name="currentText">
           <string/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <item>
           <property name="text">
            <string>Male</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Female</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="reflabel">
          <property name="text">
           <string>Reference images</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="refList">
          <item>
           <property name="text">
            <string>0_10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1_6</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetFilter">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reset filter / View all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="refImgLabel">
      <property name="minimumSize">
       <size>
        <width>460</width>
        <height>530</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="rtgImgLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openFileButton">
        <property name="text">
         <string>Open RTG image file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="boneAgeLabel">
        <property name="text">
         <string>Bone age assesment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="numberQuestionLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Number question</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="numberQuestionCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imageQuestion">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Image question</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLabel" name="twImg4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>images/patterns/trapezium/4.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="twImg5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>images/patterns/trapezium/5.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="twImg1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>images/patterns/trapezium/1.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="twAnswer1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="twAnswer5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>5</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="twAnswer4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>4</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="twImg8">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>images/patterns/trapezium/8.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="twImg3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>images/patterns/trapezium/3.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="twImg7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>images/patterns/trapezium/7.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="twAnswer3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="twAnswer7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>7</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="twAnswer8">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>8</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="twAnswer2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="twAnswer6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="twImg6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>images/patterns/trapezium/6.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="twImg2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>images/patterns/trapezium/2.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>633</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
